modules/git/commit.go

```diff
@@ -466,15 +466,15 @@ func (c *Commit) GetSubModule(entryname string) (*SubModule, error) {
 	return nil, nil
 }
 
-// GetBranchName gets the closes branch name (as returned by 'git name-rev')
+// GetBranchName gets the closes branch name (as returned by 'git name-rev --name-only')
 func (c *Commit) GetBranchName() (string, error) {
-	data, err := NewCommand("name-rev", c.ID.String()).RunInDirBytes(c.repo.Path)
+	data, err := NewCommand("name-rev", "--name-only", c.ID.String()).RunInDirBytes(c.repo.Path)
```

Comment:
As we're just using result as a string may as well do the string conversion here.

```suggestion
	data, err := NewCommand("name-rev", "--name-only", c.ID.String()).RunInDir(c.repo.Path)
```